src/styles/components/substring-highlight.scss

```
@@ -0,0 +1,11 @@
+@import '../colours';
+
+.highlight {
+  font-weight: bold;
+  background-color: rgba(
+    red($colour-medium-turquoise),
+    green($colour-medium-turquoise),
+    blue($colour-medium-turquoise),
+    0.4
+  );
```

I think I've used something like
```suggestion
  background-color: rgba(
    $colour-medium-turquoise,
    0.4
  );
```
in the past and I believe it should work too